Ported/combat-bees/Assets/Scripts/Authoring/NestAuthoring.cs

```diff
@@ -16,5 +17,6 @@ public override void Bake(NestAuthoring authoring)
         MeshRenderer mesh = GetComponent<MeshRenderer>();
         AddSharedComponent<Faction>(new Faction{Value = (int)authoring.factions});
         AddComponent<Area>(new Area{ Value = AABBExtensions.ToAABB(mesh.bounds) });
+        AddComponent<URPMaterialPropertyBaseColor>(new URPMaterialPropertyBaseColor{Value = (Vector4)mesh.sharedMaterial.color});
```

Comment:
@hmilletUnity : FYI. Maybe related to what you are doing right now with per-nest color determination for the spawn requests...